<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTableRenameDialog</class>
 <widget class="QWidget" name="ProjectTableRenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background: white;}
.QWidget {border-radius: 8px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>32</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="captionLabel">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black</string>
       </property>
       <property name="text">
        <string>Caption</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>11</width>
         <height>11</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>11</width>
         <height>11</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/resource/icons/icon_cross.png</normaloff>:/resource/icons/icon_cross.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>11</width>
         <height>11</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0">
     <item row="2" column="1">
      <widget class="QLabel" name="statusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: gray</string>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DialogLineEditEx" name="titleWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="statusCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DialogLineEditEx</class>
   <extends>QWidget</extends>
   <header location="global">ui/controls/dialog_line_edit_ex.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
